src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-create-or-edit.ts

```diff
@@ -0,0 +1,25 @@
+import { NextFunction, Request, Response } from "express";
+import { GitHubServerApp } from "models/github-server-app";
+
+export const JiraConnectEnterpriseAppCreateOrEdit = async (
+	req: Request,
+	res: Response,
+	next: NextFunction
+): Promise<void> => {
+	try {
+		req.log.debug("Received Jira create or edit app page request");
+
+		const uuid = req.params.uuid;
+
+		const app = uuid ? await GitHubServerApp.findForUuid(uuid) : null;
```

Comment:
> const app = await GitHubServerApp.findForUuid(uuid) should do. findForUuid should look to see if uuid is defined, if not should return null

@mboudreau This method returns the manual app form view for 2 cases- Adding a new app or Updating an existing app. So, doing a `uuid` check, if there is a `uuid` passed, it is treated as an update, if nothing's passed, will be treated as a new form.